From 30315efb159f33ab73d9842b1cc88fdba524a2e9 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 23 Oct 2012 18:06:20 +0800
Subject: [PATCH 083/602] ENGR00227841 build: optimize compile path

Change gps's ingsvcd from "system/etc/gps" to "system/bin".

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx53_smd/init.freescale.rc | 2 +-
 imx6/sabresd/init.rc        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/imx53_smd/init.freescale.rc b/imx53_smd/init.freescale.rc
index fe61d74..816c0eb 100755
--- a/imx53_smd/init.freescale.rc
+++ b/imx53_smd/init.freescale.rc
@@ -84,7 +84,7 @@ service magd /system/bin/magd
     oneshot
 
 # Orion InG
-service ing /system/etc/gps/ingsvcd -c /system/etc/gps/Orion.ini
+service ing /system/bin/ingsvcd -c /system/etc/gps/Orion.ini
     socket athrkv9988    stream 666 system system
     socket athrshmsocket stream 666 system system
     socket athrsupl59991 stream 666 system system
diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 28f04b2..03bd13c 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -106,7 +106,7 @@ service magd /system/bin/magd
     oneshot
 
 # Orion InG
-service ing /system/etc/gps/ingsvcd -c /system/etc/gps/Orion.ini
+service ing /system/bin/ingsvcd -c /system/etc/gps/Orion.ini
     socket athrkv9988    stream 666 system system
     socket athrshmsocket stream 666 system system
     socket athrsupl59991 stream 666 system system
-- 
1.8.0

